superset/assets/src/components/AnchorLink.jsx

```diff
@@ -40,24 +43,32 @@ const defaultProps = {
 class AnchorLink extends React.PureComponent {
 
   componentDidMount() {
-    const hash = this.getLocationHash();
+    const hash = getLocationHash();
     const { anchorLinkId } = this.props;
 
     if (hash && anchorLinkId === hash) {
-      const directLinkComponent = document.getElementById(anchorLinkId);
-      if (directLinkComponent) {
-        setTimeout(() => {
-          directLinkComponent.scrollIntoView({
-            block: 'center',
-            behavior: 'smooth',
-          });
-        }, 1000);
-      }
+      this.scrollToView();
     }
   }
 
-  getLocationHash() {
-    return (window.location.hash || '').substring(1);
+  componentWillReceiveProps(nextProps) {
+    const { inFocus = false } = nextProps;
+    if (inFocus) {
+      this.scrollToView(0);
```

Comment:
this doesn't need an argument right? or you can remove the default arg on the function

Reply:
fixed.